src/test/java/com/artipie/pypi/PypiITCase.java

```diff
@@ -23,205 +23,85 @@
-                new Key.From("pypi-repo", "example-pckg")
-        );
-        this.cntn.execStdout("python3", "-m", "pip", "install", "--user", "--upgrade", "twine");
-        MatcherAssert.assertThat(
-            "Packages should be uploaded",
-            this.cntn.execStdout(
-                "python3", "-m", "twine", "upload", "--repository-url", this.url.string(),
-                "-u", ArtipieServer.ALICE.name(), "-p", this.pswd(anonymous),
-                "pypi-repo/example-pckg/*"
-            ),
-            new StringContainsInOrder(
-                new ListOf<String>(
-                    "Uploading artipietestpkg-0.0.3-py2-none-any.whl", "100%",
-                    "Uploading artipietestpkg-0.0.3.tar.gz", "100%"
-                )
+    @RegisterExtension
+    final TestDeployment containers = new TestDeployment(
+        () -> TestDeployment.ArtipieContainer.defaultDefinition()
+            .withRepoConfig("pypi-repo/pypi.yml", "my-python"),
+        () -> new TestDeployment.ClientContainer("python")
+            .withWorkingDirectory("/var/artipie")
+            .withClasspathResourceMapping(
+                "pypi-repo/alarmtime-0.1.5.tar.gz",
+                "/var/artipie/data/artipie/pypi/alarmtime/alarmtime-0.1.5.tar.gz",
+                BindMode.READ_ONLY
+            )
+            .withClasspathResourceMapping(
+                "pypi-repo/example-pckg",
+                "/var/artipie/data/artipie/pypi/example-pckg",
+                BindMode.READ_ONLY
             )
+    );
+
+    @BeforeEach
+    void setUp() throws IOException {
+        this.containers.assertExec(
+            "Apt-get update failed",
+            new MavenITCase.ContainerResultMatcher(),
+            "apt-get", "update"
         );
-        MatcherAssert.assertThat(
-            "Packages should be saved in storage",
-            this.inStorage("artipietestpkg-0.0.3-py2-none-any.whl")
-                && this.inStorage("artipietestpkg-0.0.3.tar.gz"),
-            new IsEqual<>(true)
+        this.containers.assertExec(
+            "Install twine",
+            new MavenITCase.ContainerResultMatcher(),
+            "python", "-m", "pip", "install", "twine"
         );
     }
 
     @Test
-    void pypiInstallShouldFailWithForbidden() throws Exception {
-        this.init(false);
-        MatcherAssert.assertThat(
-            this.cntn.execStdoutWithoutCheckExitCode(
-                "pip", "install", "--verbose", "--no-deps", "--trusted-host", PypiITCase.HOST,
-                "--index-url", this.url.string(ArtipieServer.BOB), "anypackage"
-            ),
-            new StringContains(
-                String.format(
-                    "403 Client Error: Forbidden for url: %spip/", this.url.string()
-                )
-            )
+    void pushAndInstallWorks() throws Exception {
+        this.containers.assertExec(
+            "Apt-get update failed",
+            new MavenITCase.ContainerResultMatcher(),
+            "apt-get", "update"
```

Comment:
@milkysoft there is no need to perform `apt-get update` again, it's already done in `BeforeEach`. Here we should: 
1) add necessary resource (alarmtime-0.1.5.tar.gz) to artipie container (use `putResourceToArtipie()` method)
2) call `pip install`

also there will not be any `push` in this method, so it should be properly renamed. We follow [these](https://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names) suggestions for test method names

Reply:
@olenagerasimova , latest commit added integrations test:
1. Setup python package with `putResourceToArtipie` onto Artipie server
2. Install python package on client container with `pip install`